### ISSUE TITLE
LocalDate::now($timezone /*= system TimeZone*/)

### ISSUE THREAD

**Issue by nkamuo** [brick/date-time#113]
The `LocalDate::new()`, `LocalTime::new()` and `LocalDateTime::new()`  all require the timezone parameter.
It would be a lot easier to use if the `::now()` method's timezone parameter is optional and defaults to the configured php TimeZone.

**Comment by zjbarg** [comment]
that's what I'm referring to in #111

**Comment by brunocassol** [comment]
> developers always make a conscious decision when selecting a timezone

I agree with this decision. Timezone is not something you want an implicit default when dealing with dates.

And applications can always define an explicit default timezone and pass that. Something like ` LocalDate::new(MY_APP_DEFAULT_TIMEZONE)`.
